pkgs/by-name/ka/kapacitor/package.nix

```
@@ -40,39 +43,41 @@ let
       })
     ];
     sourceRoot = "${src.name}/libflux";
-    cargoHash = "sha256-oAMoGGdR0QEjSzZ0/J5J9s/ekSlryCcRBSo5N2r70Ko=";
+    cargoHash = "sha256-yIYeJvLe+L72ZyuQ2AK6l4HGSF/tgCyGQsXEOWUXDn0=";
     nativeBuildInputs = [ rustPlatform.bindgenHook ];
     buildInputs = lib.optional stdenv.hostPlatform.isDarwin libiconv;
     pkgcfg = ''
       Name: flux
-      Version: ${libflux_version}
+      Version: v${libflux_version}
```

I don't think this is correct. All pkg-config files I have checked do not have a prefix like this.

```suggestion
      Version: ${libflux_version}
```